test/system/groups/samples_test.rb

```diff
@@ -16,6 +16,16 @@ def setup
       @sample31 = samples(:sample31)
     end
 
+    def retrieve_puids
+      within first('table tbody#group-samples-table-body') do
+        puids = []
+        puids << first('tr:nth-child(1) td:first-child').text
```

Comment:
**suggestion:** Simplify this by putting it in a loop

Reply:
Changed in a5a08729ba05d4d32eaf14bd0833e445db4b336f